awx_collection/plugins/modules/workflow_job_template.py

```diff
@@ -736,11 +750,12 @@ def create_workflow_nodes_association(module, response, workflow_nodes, workflow
                 'failure_nodes',
                 'credentials',
                 'labels',
+                'inventory',
                 'instance_groups',
             ):
                 # Extract out information if it exists
                 # Test if it is defined, else move to next association.
-                prompt_lookup = ['credentials', 'labels', 'instance_groups']
+                prompt_lookup = ['credentials', 'labels', 'inventory', 'instance_groups']
```

Comment:
This seems suspicious since these are all plural, many-to-many, relationships but inventory is not.

Reply:
It's derived from the related field
![image](https://user-images.githubusercontent.com/30054029/219692326-50f92dce-8e01-47be-9bc4-ad31738c2221.png)